### PR TITLE
fix(alerts): avoid AttributeError in EventAttributeCondition

### PR DESCRIPTION
**Pull request by cathteng**
Fixes SENTRY-3H0X

### CODE REVIEW

File: src/sentry/rules/conditions/event_attribute.py [hunk 1]
```diff
@@ -256,7 +256,7 @@ def _handle(cls, path: list[str], event: GroupEvent) -> list[str]:
             return []
 
         return [
-            getattr(e, path[1])
+            str(getattr(e, path[1], None))
```
Comment (ceorourke) [inline src/sentry/rules/conditions/event_attribute.py:259]: Why is this one stringified?

Comment (cathteng) [reply src/sentry/rules/conditions/event_attribute.py:259]: mypy gets angry

Comment (ceorourke) [reply src/sentry/rules/conditions/event_attribute.py:259]: Weird, could we run into a problem later on with a string of `'None'` though?

Comment (cathteng) [reply src/sentry/rules/conditions/event_attribute.py:259]: replaced with hasattr

**Review by ceorourke — APPROVED** [review]

**Pull request merged**